_posts/en/pages/2016-01-01-about.md

```diff
@@ -10,11 +10,15 @@ version: 1
 redirect_from:
   - /zh_TW/about/
 ---
-
 Bitcoin Core is an [open source](https://opensource.org/) project which maintains and releases Bitcoin client software called "Bitcoin Core".
 
 It is a direct descendant of the original Bitcoin software client released by Satoshi Nakamoto after he published the famous [Bitcoin whitepaper](/bitcoin.pdf).
```

Comment:
Is Satoshi Nakamoto confirmed to be singular male, i.e. "he"?

Reply:
I believe he presented this way. Yes. And I can probably find that reference, if it's important. But I would politely suggest that since 'he' wasn't a change I submitted, perhaps that's not germane to the PR?